go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -45,6 +46,9 @@ import (
 	vtrpcpb "github.com/youtube/vitess/go/vt/proto/vtrpc"
 )
 
+// TODO(sougou): remove after affected parties have transitioned to new behavior.
+var legacyTableACL = flag.Bool("legacy-table-acl", false, "deprecated: this flag can be used to revert to the older table ACL behavior, which checked access for at most one table")
```

Comment:
Could you change the default to true?

If set to false and someone creates a rollout without setting the flat to false then they'd roll out the new behavior. And we can't set the flag in our builds until it's present in the code. We can change the default after we've done the rollout.

Reply:
We can solve that with undefok, right?
Whichever way we go, we'll see what's more convenient for us and we can fix it in the tree...

Reply:
I agree with Alain; We should keep the default as false and use undefok to control the rollout of the new behavior.